src/components/Person/Person.jsx

```diff
@@ -1 +1,22 @@
-// export const Person = ({ person }) => ();
+export const Person = ({
+  person,
+  partner = person.sex === 'm' ? 'wife' : 'husband',
```

Comment:
But it's not a prop, next time move component variables below and add return statement to render html

Reply:
you mean 
export const Person = ({
  person,
  partner = person.sex === 'm' ? 'wife' : 'husband';
  
  return (...)
  });?